core/app.go

```diff
@@ -59,14 +64,17 @@ func EmptyApp() *App {
 }
 
 // LoadApp parses the commandline arguments and loads the config
-func LoadApp() (*App, error) {
+func LoadApp() (Runnable, error) {
 
 	var configFlag string
+	var renderFlag string
 	var versionFlag bool
 
 	if !flag.Parsed() {
 		flag.StringVar(&configFlag, "config", "",
 			"JSON config or file:// path to JSON config file.")
+		flag.StringVar(&renderFlag, "render", "",
+		  "- for stdout or file:// path where to save redenred JSON config file.")
```

Comment:
Sorry, 'flag' did not work as I thought it would. I am not sure how to implement what you asked without bigger surgery.